matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
```

Comment:
There is a marginal improvement in time (due to parsing arguments) due to
optimised mode. A getState and setState function are all that is needed in
my opinion. We can eliminate setWorldFrame for sure though.

On Thu, Oct 29, 2015 at 11:24 AM, Silvio Traversaro <
notifications@github.com> wrote:

> In matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m
> <https://github.com/robotology/mex-wholebodymodel/pull/52#discussion_r43368831>
> :
>
> > @@ -0,0 +1,33 @@
> > +function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
>
> "Normal mode" is the state-less interface (where you provide every time
> the state to the functions) while "optimized mode" is the stateful
> interface (where you provide the state once with
> wbm_setWorldFrame/wbm_updateState and then you use that one), right?
>
> If you always use the stateless/normal mode interface, I think you can
> remove both wbm_setWorldFrame and wbm_updateState, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robotology/mex-wholebodymodel/pull/52/files#r43368831>
> .
>